New functions (#195)
aoijs/website: src/content/docs/functions/newEntitlementData.md

---
title: $newEntitlementData

> description: $newEntitlementData will return the old entitlement data of the provided SKU.

Reviewer: ```suggestion
description: $newEntitlementData will return the old entitlement data.
```

id: newEntitlementData
---

> `$newEntitlementData` will return the new entitlement data of the provided SKU.

Reviewer: ```suggestion
`$newEntitlementData` will return the new entitlement data.
```

## Usage

```aoi

> $newEntitlementData[skuID]

Reviewer: ```suggestion
$newEntitlementData[option]
```

```

## Parameters

| Field         | Type                                                                                                | Description                                                                    | Required |
| ------------- | --------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------ | :------: |

> | skuID         | [number](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number)   | Your generated [SKU ID](https://discord.com/developers/docs/resources/sku).    |   true   |

Reviewer: ```suggestion
| option         | [string](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number)   | The data to return    |   true   |
```
